### ISSUE TITLE
How does DICSS compare to DILDOSS?

### ISSUE THREAD

**Issue by MadaraUchiha** [letsgetrandy/DICSS#44]
A friend had compared DICSS to DILDOSS (Dynamically Injected Lexically Documented and Organized StyleSheets), I didn't check it myself, but she says DILDOSS is much more stable and predictable. Moreover, scale appears to be easier with DILDOSS.

**Comment by bkoifman** [comment]
Most sure that using DILDOSS can be a pain in the ass, yet still much more stable

**Comment by letsgetrandy** [comment]
Unfortunately, while DILDOSS is more stable, it's never _not_ hard to use.

**Comment by bkoifman** [comment]
DILDOSS also resolved the leaks that DICSS has, which make it more secure and should keep you out of trouble for the next few years

**Comment by letsgetrandy** [comment]
True. With DICSS, depending on your implementation, there is the risk of spawning a new processes, which is not possible with DILDOSS.
